### PR TITLE
First try Adding paintings and xeno crystals, please be gentle.

### PR DESCRIPTION
**Pull request by Dizor**
Changes old Xenoarch Green and Pink crystals to the ones I updated. Also added in the paintings I did to see if it works. Hopefully I didnt fuck up but you know.

paintings show

paintings show2
Temmie and Oblivion Paintings not included. Only carp added
xenoarchpv

### CODE REVIEW

**Comment by clusterfack** [comment]
Pictures please

**Comment by Dizor** [comment]
Pics added

**Comment by clusterfack** [comment]
Neat

**Comment by Zth--** [comment]
Love them

**Comment by DeityLink** [comment]
Oh christ how come I missed that. Yeah no wonder the new paintings don't work.

File: code/game/objects/items/mountable_frames/paintings.dm [hunk 1]
```diff
@@ -76,6 +90,49 @@ var/global/list/available_paintings = list(
...
+			desc = "There are no words to discribe this painting."
+		if("wizard")
+			name = "\improper Nausiating Glow in the Dark Velvet Wizard Poster"
+			desc = "Oh god he's looking right at me, what do I do what do I do!?"
+		if("bland")
+			name = "\improper Pitcher and Orange"
+			desc = "A painfully standard painting, used to decorate dining rooms and bathrooms alike."
+		if("Blu")
+			name = "\improper Wai-Blu"
+			desc = "Faithfully Serving NanoTransen during her shift, gladly serving YOU after."
```
Comment (BestName) [inline code/game/objects/items/mountable_frames/paintings.dm:119]: > NanoTransen

Fuck you

File: code/game/objects/items/mountable_frames/paintings.dm [hunk 2]
```diff
@@ -76,6 +90,49 @@ var/global/list/available_paintings = list(
...
+		if("Kate")
+			name = "\improper Cindy Kate"
+			desc = "Through the carnage and bloodshed she's gunning for you, champ."
+		if("daddy")
+			name = "\improper I <3 Daddy!"
+			desc = "'NanoTransen respects the right for all associates and their families to be able to express their indivuality though many media. However, soliciting NanoTransen related merchandise without proper warrant is strickly prohibited. Luckly for you, you can now own your very own contraband NanoTransen merch without the threat of *REDACTED*!'"
```
Comment (BestName) [inline code/game/objects/items/mountable_frames/paintings.dm:125]: Double fuck you!

File: code/game/objects/items/mountable_frames/paintings.dm (hunk 1 above)
Comment (clusterfack) [inline code/game/objects/items/mountable_frames/paintings.dm]: Thats rude desu

Comment (BestName) [inline code/game/objects/items/mountable_frames/paintings.dm:119]: I don't care! I've been triggered!

Comment (DeityLink) [inline code/game/objects/items/mountable_frames/paintings.dm:119]: Can't you just as quickly make a PR to fix it?

Comment (BestName) [inline code/game/objects/items/mountable_frames/paintings.dm:119]: But I did?! #8058

**Pull request merged**